Lib/test/test_capi/test_long.py

```diff
@@ -211,9 +211,9 @@ def check_long_asintandoverflow(self, func, min_val, max_val):
 
         self.assertEqual(func(min_val - 1), (-1, -1))
         self.assertEqual(func(max_val + 1), (-1, +1))
+        self.assertRaises(SystemError, func, None)
 
         # CRASHES func(1.0)
```

Comment:
Does it really crash?

Reply:
Good catch, it doesn't: "If any other exception occurs set *overflow to 0 and return -1 as usual."  TypeError will be here.